packages/iobio-react-components/README.md

```diff
@@ -2,14 +2,52 @@
 
 ## Setup + Local Development
 
+### Requirements:
+
+- PNPM 9+
+- Node 20+
+
 - Install dependencies: `pnpm i`
 
-- Import components in consumer app as `import IobioComponents from '@overture-stack/iobio-components/components/src/index';`
+- Import components in consumer app as `import { IobioDataBroker } from '@overture-stack/iobio-components/packages/iobio-react-components/;`
+
+### Build Package
+
+- `pnpm run build`
 
 ## React Components
 
-- Stub for React Component documentation.
-<!-- TODO: Write Docs (include pnpm, monorepo, etc) -->
+- This package exports a set of React + TypeScript components to facilitate the integration of the Iobio Charts library of "web components" into your UI app.
+
+- The Demo App at `iobio-components/apps/bam.iobio` shows an example integration which closely mirrors the integration at bam.iobio.io
+
+- The `<IobioDataBroker />` component is necessary to analyze a given file. It receives an `alignmentUrl` string prop which should point to a BAM/CRAM file URL. The Iobio server performs an analysis on the given file, and the other Iobio components read this data.
+
```

Comment:
I think `..and the other Iobio components read this data` could be expanded on. It's not immediately clear or a super common that the React components are mostly wrapping web components for display, which are hooked in to events in the data broker components. If I came across this repo, without seeing it being developed, I would assume React (or React compatible libraries) were controlling the state.

Reply:
OK added some clarified language